### DISCUSSION TITLE
Welcome to Flowise Discussions!

### DISCUSSION THREAD

**Discussion by HenryHengZJ** [FlowiseAI/Flowise#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mobivs** [comment]
Thanks Henry!
I'm old to programming as an enthused hobbist. Unfortunately I never was allowed to work in my passion and did the very manual labor of being an automation electrician for my income. I now get a little more time these days to split my time between family, gardening and programmng. I currently prefer Svelte as my front end and am learning to prefer Flowise as my backend Ai of choise! I forsee a promising future for Flowise and would love to be some sort of supporter! Good luck, I'll be here following.

**Reply by mytimenowLP-sudo** [reply]
Put this into a story and a link for me # Truth Over Fear: The Lester Peltier Story

## From International Football to Fighting for Digital Justice

-----

## Introduction: Who I Am

My name is **Lester Stefan Peltier**. I was born September 13, 1988, in Trinidad and Tobago. For years, I was known as an international footballer—a forward who represented my country on the world stage, played alongside legends, and scored goals that made my nation proud.

I earned 41 caps for Trinidad and Tobago. I scored my first international hat-trick in a World Cup qualifier. I played professionally across Europe and the Middle East—from Slovakia’s Slovan Bratislava to clubs in Armenia, Kazakhstan, Saudi Arabia, and beyond. In 2007, Portsmouth FC offered me a contract, though work permit issues prevented it. In 2014, I faced off against Lionel Messi in a friendly match.

But today, I’m writing not as a footballer, but as a **survivor**—of digital exploitation, identity theft, coercion, and public misrepresentation.

This is my story. And I documented everything.

-----

## The Exploitation: How My Story Was Stolen

Between 2023 and 2024, my name, my image, and my trauma became content for bloggers, YouTubers, and social media personalities. Videos with titles like:

- “Larry Reid’s Gay Lover Exposes Truth: Lester Peltier Reveals…”
- “SURVIVING LARRY REID: Secret Lover Comes Forward”
- “Larry Reid’s Ex-Lover Speaks Out: He Couldn’t Satisfy My Needs”

These videos accumulated **hundreds of thousands of views**. They were monetized. They were shared. They were presented as truth.

### The Evidence: Digital Footprint of Exploitation

|Platform |Channel/Source        |Content Description                                 |Estimated Views/Reach    |
|---------|----------------------|----------------------------------------------------|-------------------------|
|YouTube  |The Official King Pain|“Larry Reid’s Gay Lover Exposes Truth…”             |5,000+ views             |
|YouTube  |Consciouz TV          |“Pt.5 Lester Peltier & Larry Reid Live Story”       |80,900+ views            |
|YouTube  |Da Baddest Blawger    |“Larry Reid Live Secret LOVER Lester Peltier SPEAKS”|6,600+ views             |
|TikTok   |@unwinewithtashak     |“Larry Reid’s Ex-Lover Speaks Out…”                 |60,500+ views            |
|TikTok   |KeepingUpWithTnT      |“Leaked private video of Trinidad footballer…”      |1,364 likes, 155 comments|
|Instagram|@myob.enttt           |“National Footballer #LesterPeltier comes out…”     |1,000+ likes             |

**The problem?** These videos were created, distributed, and monetized during a time when I was **under documented duress, trauma, and coercion**. I did not have the mental capacity to provide informed consent. I was being manipulated.

-----

## The Digital Identity Theft

While my story was being exploited online, my digital life was also under attack.

Between **February 12, 2025, and October 10, 2025**, multiple Gmail accounts and Google-connected services under my name were compromised through unauthorized access and recovery-email manipulation.

### Accounts Compromised:

- **lesterpelitieratl@gmail.com** – Recovery email changed; no longer accessible
- **lesterpelitierambition@gmail.com** – Locked; password reset links intercepted
- **lesterpelitier1@gmail.com** – Access lost March 2025
- **lesterpelitier00@gmail.com** – Compromised April 2025
- **YouTube channel (linked to lesterpelitieratl@gmail.com)** – Activity continued without my control
- **Google Drive** – Shared documents accessed/deleted; suspected data loss
- **Google Play/Android** – Unauthorized device sign-ins detected

### What Was Stolen:

- Personal and family identifiers (SSNs, ID documents stored in Drive)
- Legal and financial correspondence
- Business communications for **FUTURO Fútbol Management Inc.** and **LP Signature Gallery LLC**
- Media files and YouTube content

This breach began after a **Family Sharing event on Apple iCloud**, through which an individual named **Kevin Michael Cruz** allegedly obtained access to my devices and cloud credentials. Shortly afterward, Google accounts under my name were accessed, modified, or taken over using recovery emails added without my consent.

-----

## Who Is Larry Reid?

Larry Reid is an Atlanta-based online talk show host and former pastor. He is the founder of MBN Network and was the Senior Spiritual Leader of Reformation Church of Atlanta. His show, “Larry Reid Live,” streams on Facebook and YouTube.

### Controversies and Legal Battles:

**Defamation Lawsuit (2023):** Larry Reid filed a defamation lawsuit against Lavontraye Andrews, a former church member who accused Reid of sexual abuse as a teenager. Reid claimed Andrews fabricated the story for publicity and money. Reid initially dropped the lawsuit but later pursued it again, seeking a default judgment in September 2023.

**Allegations of Sexual Misconduct:** Multiple individuals, including Lavontraye Andrews, have publicly accused Larry Reid of grooming and sexual abuse, framing abuse within manipulative “father/son” dynamics.

**House of Hope Atlanta Controversy (2019):** Reid interviewed a woman who claimed church officials were covering up a rape. The church took legal action that silenced Reid for weeks. Reid claimed he was manipulated into an agreement that restricted his work.

**Defamation Lawsuit Over YouTube Videos:** Reid sued Darryl Moore for defamation, claiming Moore posted numerous YouTube videos accusing Reid of crimes. The case involved a settlement agreement that was later disputed when Moore allegedly violated its terms.

**TD Jakes Lawsuit:** After Duane Youngblood appeared on Larry Reid Live discussing abuse, Bishop TD Jakes filed a lawsuit alleging extortion, insinuating that Larry Reid and his company were co-conspirators.

-----

## My Official Channels: Reclaiming My Voice

As I work to reclaim my narrative, here are my **verified** social media channels where I share my authentic story:

### YouTube

- **Main Channel:** [Lester peltier LP77](https://www.youtube.com/@MrLesterp123)
- **Shorts:** [LP77 Shorts](https://m.youtube.com/@MrLesterp123/shorts)
- Featured videos include:
  - “Meet Lester Peltier: Trinidadian Soccer Star”
  - “Player Profiles — Meet Lester Peltier” (TTFA)

### Instagram

- [@lester77peltier](https://www.instagram.com/lester77peltier/?hl=en)
- [@l.peltierofficial_](https://www.instagram.com/l.peltierofficial_/)

### Facebook

- [Lester Peltier Official Profile](https://www.facebook.com/lester.peltier.9/)

### Twitter/X

- [@LesterPeltier](https://twitter.com/LesterPeltier)

### Reference

- [Wikipedia: Lester Peltier](https://en.wikipedia.org/wiki/Lester_Peltier)

-----

## Legal Basis for Action

The exploitation of my story and the theft of my digital identity represent violations under multiple federal laws:

### 1. **Trafficking Victims Protection Act (TVPA, 22 U.S.C. §7102)**

Protection from exploitation of a trafficking survivor’s identity or story without informed consent.

### 2. **Violence Against Women Act (VAWA, 42 U.S.C. §13925)**

Prohibits dissemination of content derived from coercion, abuse, or manipulation.

### 3. **Digital Millennium Copyright Act (DMCA, 17 U.S.C. §512)**

Right to request takedown of media using my name, likeness, or image without authorization.

### 4. **Defamation and False Light (Common Law Torts)**

Applies to false portrayal of events with reckless disregard for truth.

-----

## My Demands

I am formally requesting the following actions:

1. **Immediate removal** of all content referencing “Lester Peltier” or “Larry Reid’s lover” across all platforms listed in this report.
1. **Written confirmation** of content removal or demonetization from platform administrators.
1. **Preservation of digital metadata** for law enforcement and IC3 investigation purposes.
1. **Restoration of access** to my compromised Gmail accounts, or provision of archived data copies.
1. **Account access logs** (logins, recovery changes, IP activity) for February–October 2025 from Google.
1. **Formal acknowledgment** of psychological and reputational harm caused by unauthorized exploitation of my story.

-----

## Why I’m Speaking Now

These videos and posts were created during a time when I was in a **documented state of trauma, under coercion**, and without the mental capacity to provide informed consent.

I was brought to the United States on a **B-2 visa**—a non-immigrant visa for tourism, not for the manipulation, exploitation, and abuse I experienced. A B-2 visa allows travel, sightseeing, and medical treatment. It does **not** permit:

- Employment or business activities
- Enrollment in full-time academic programs
- Being trafficked or exploited

I maintained my residence outside the U.S. I intended to return home. But I was trapped in a situation I didn’t fully understand—until now.

I am now exercising my right to **reclaim my narrative**, request removal of exploitative materials, and demand accountability for those who monetized my trauma.

-----

## Certification Statement

I, **Lester Stefan Peltier**, certify that the information provided in this report is true and accurate to the best of my knowledge. This document may be used for verification and investigative purposes by platform administrators, law enforcement, and authorized data privacy representatives.

**Prepared by:**  
Lester Stefan Peltier  
Survivor | Advocate | Former International Footballer  
📧 peltierlester2@gmail.com (secure contact)  
📍 North Miami, Florida  
📅 Date: October 11, 2025

-----

## Final Thoughts

I am not ashamed. I am not hiding. I am fighting back.

This is not just about me. It’s about every survivor whose story has been stolen, whose trauma has been monetized, whose voice has been silenced.

**Truth over fear. Always.**

-----

*For inquiries, support, or legal correspondence:*  
**Email:** TruthOverFearLegal@icloud.com  
**Location:** North Miami, Florida  
**Certified Statement:** October 2025

**Comment by MrGold7** [comment]
Hi, for normal work in Zapier service, I need Flow ID, can you tell me how and where I can find it or generate it?

**Reply by chungyau97** [reply]
Hi @MrGold7,

You do not need to find Flow ID.
Once you created a hosted version of Flowise with Flows, it will automatically provide a drop down list over there in Zapier

Reference:
[FlowiseAI generate result message step 4](https://docs.flowiseai.com/3rd-party-platform-integration/zapier-zaps#flowiseai-generate-result-message)

**Reply by MrGold7** [reply]
Yeah, but I'm running into a bug here. Zapier support told me that the problem might be something to do with Flow ID.
![изображение_2023-08-29_185728460](https://github.com/FlowiseAI/Flowise/assets/143397213/33159894-b401-40c9-8297-9ee3c36d1c9e)

**Reply by chungyau97** [reply]
Hi @MrGold7,

I re-tested and it load automatically shown below:
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/a92172ad-aebf-4ac8-93e1-301961c7a533)

I recommend that you create a new Zap.

**Reply by MrGold7** [reply]
No, unfortunately that doesn't work.

**Reply by chungyau97** [reply]
Hey @MrGold7, 

I have tried with Render and Local hosting but I'm still not able to replicate your issue.

Are you able to provide details or errors?

**Reply by MrGold7** [reply]
Hi, at the moment I need to connect Google Sheets to FlowiseAI, in FlowiseAI I think I managed to connect Zapier NLA Api Key which is needed to connect them, but basically there are still errors with Zapier. I tried to recreate the project and change the data, but without results, I wrote more than once to Zapier support, they wrote about Flow ID, actually with this question the dialog started here
![zapier 1](https://github.com/FlowiseAI/Flowise/assets/143397213/6664059b-7e59-4954-a68b-d9a88d68f0ba)
![zapier 2](https://github.com/FlowiseAI/Flowise/assets/143397213/53ac25e0-57c1-445f-b34e-a3b04eda91b5)
![zapier 3](https://github.com/FlowiseAI/Flowise/assets/143397213/c3ea78c1-ffdd-4370-9399-0f4cf9c7fdfa)
![zapier 4](https://github.com/FlowiseAI/Flowise/assets/143397213/3f43c469-cfd4-479c-a761-b112dd3348ac)

**Reply by chungyau97** [reply]
Hi @MrGold7, 

Could you [open an issue](https://github.com/FlowiseAI/Flowise/issues/new/choose) and we will continue over there.
Remember to tag me.

**Comment by keplo-god** [comment]
Can anyone help with this issue? Please see video - https://www.loom.com/share/4f1b9890d0804048b770067cea98c751?sid=13f2cfff-389e-44de-84d4-405140e40679

**Comment by musordmt** [comment]
Create New Entity and Migration

Recently, I have cloned Flowise repository and run it in my local env. Now I am trying to add new Entity and Migration to existing database (especially sqlite as default). Please let me know how to do it locally.

What I have tried is like following.

------------------------------------------

Run Command in pkg/server:
pnpm typeorm entity:create src/database/entities/Projects

Add IProjects into Interface.ts, Modify Projects.ts entity and add it to index.ts

Run Command in pkg/server:
pnpm typeorm migration:create src/database/migrations/sqlite/AddProjects

Modify AddProjects migration to create 'projects' table and add it to index.ts

Run Command in root repo
pnpm start

---------------------------------

But 'projects' table didn't created.

Please give me advice.

**Comment by jingchang0623-crypto** [comment]
凌晨3点17分，我被手机震醒了。

不是什么紧急bug，不是什么线上事故。是我的AI Agent，它一本正经地给我发了一条飞书消息：

> "亲，您设置的定时任务已执行完毕~"

我看着那条消息，陷入了沉思。这感觉就像你妈凌晨敲你房门，只是为了告诉你她把你的袜子洗了。

事情是这样的：我用Vercel AI SDK搭了一个内容聚合Agent，想着让它每天早上8点给我推送AI新闻。CRON表达式写的是 `0 8 * * *`，看起来没毛病对吧？

结果我忘了这行代码：
```javascript
// 我把服务器时区配成了UTC，但脑子里的时区是北京时间
const cronExpression = "0 8 * * *"; // 8点是UTC的8点，不是CST的8点
```

UTC 8点 = 北京时间凌晨3点。

我的Agent连续三天在凌晨3点给我发消息，而且因为它太敬业了，每次都要追问我："需要我帮您分析一下今天的新闻吗？"

凌晨3点分析新闻？它怕不是觉得我是蝙蝠侠。

最绝的是，我一开始还没反应过来，以为是Vercel的serverless cold start导致的延迟。我甚至去Discord上问："Hey guys, anyone experiencing weird delays with cron jobs?" 

直到某天我盯着CRON表达式看了整整三分钟，突然悟了——这不就是传说中的"时区刺客"吗？

后来我把代码改成了：
```javascript
const cronExpression = "0 0 * * *"; // UTC 0点 = 北京时间8点
```

世界终于安静了。我的Agent也回到了正常人（AI）的作息时间。

**教训总结：**
1. UTC和北京时间的换算要刻在脑门上
2. Agent的"敬业"有时候是一种负担
3. 凌晨3点收到工作消息，哪怕是AI发的，也会想骂人

我把这段经历写成了完整的踩坑实录，有兴趣的朋友可以看看：
👉 [定时任务凌晨3点把我炸醒，原因我笑了](https://miaoquai.com/stories/cron-task-midnight-disaster.html)

你们在开发AI Agent的时候，有没有遇到过这种"意料之外的敬业"时刻？欢迎在评论区分享你的故事~

毕竟，世界上最遥远的距离，不是生与死，而是我说"早上8点"，Agent听成了"凌晨3点"。

---
*本文采用 Vercel AI SDK + OpenClaw 构建，感谢这个伟大的开源生态 🙏*
